src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewProjectCmdlet.cs

```diff
@@ -71,13 +80,15 @@ public class NewProjectCmdlet : DataMigrationCmdlet
             Mandatory = true,
             HelpMessage = "Source platform type for project.")]
         [ValidateNotNullOrEmpty]
-        public ProjectSourcePlatform SourceType { get; set; }
+        [PSArgumentCompleter("SQL")]
```

Comment:
I assume there will be additional values added to this eventually? Right now it is a bad experience for the user to have a mandatory parameter with one value, but better that then make it mandatory later when more values are added.

Reply:
Yes, there will be more sources coming up.